### PR TITLE
Add Dynatrace to landscape model

### PR DESCRIPTION
**Pull request by mreider**
Added Dynatrace, OneAgent, ActiveGate, and Micrometer to the OpenAPM landscape.

### CODE REVIEW

File: model/components/commercial/dynatrace.yml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        - tracing
+        - metrics
+        - logs
+    connections:
+      includesFrom:
+        - opentelemetry-format
+        - prometheus-exposition-format
+      dataTo:
+        - grafana
+      dataFrom:
```
Comment (arile01) [inline model/components/commercial/dynatrace.yml:29]: influx, prometheus and jaeger need to actually reference another component id. So e.g. 'influxdata-influxdb' or 'influxdata-telegraf' instead. Which id's are available can be looked up in the respective component file yaml.

File: model/components/commercial/dynatrace.yml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        - metrics
+        - logs
+    connections:
+      includesFrom:
+        - opentelemetry-format
+        - prometheus-exposition-format
+      dataFrom:
+        - opencensus
+        - opentelemetry
+        - prometheus
```
Comment (arile01) [inline model/components/commercial/dynatrace.yml]: same here :)

**Review by arile01 — CHANGES_REQUESTED** [review]
Hey there, there are some unresolved references which need to be fixed as the data-model cannot be build otherwise. :) 
You can check if the model can be build with >$npm run-script build <

**Comment by mreider** [comment]
I think these satisfy your comments?

**Pull request merged**